plugin/wintabs.vim

```diff
@@ -71,7 +71,8 @@ call s:set('g:wintabs_ui_sep_inbetween', '|')
 call s:set('g:wintabs_ui_sep_rightmost', '|')
 call s:set('g:wintabs_ui_active_left', ' ')
 call s:set('g:wintabs_ui_active_right', ' ')
-call s:set('g:wintabs_ui_show_vimtab_name', 0)
+call s:set('g:wintabs_ui_name_format', '%n')
+call s:set('g:wintabs_ui_vimtab_name_format', '%d')
```

Comment:
to be backward-compatible, let's condition on the value of `g:wintabs_ui_show_vimtab_name` here to decide the default value of `g:wintabs_ui_vimtab_name_format` (`0 => '%d', 1 => '%n', 2 => '%d:%n'`).